README.md

```diff
@@ -3,40 +3,114 @@
 
-Red Hat Advanced Cluster Management Governance - Policy Framework
+Open Cluster Management - Governance Policy Framework
 
 The policy framework provides governance capability to gain visibility and drive remediation for various security and configuration aspects to help meet such enterprise standards.
 
 ## What it does
 
 View the following functions of the policy framework: 
 
-* Applies policies to managed clusters from hub cluster.
+* Distributes policies to managed clusters from hub cluster.
 * Collects policy execution results from managed cluster to hub cluster.
+* Supports multiple policy engines and policy languages.
 * Provides an extensible mechanism to bring your own policy.
 
 ## Architecutre
 
-The policy framework consists of following components:
-
-- [Governance dashboard](https://github.com/open-cluster-management/grc-ui)
-- [Policy propagator](https://github.com/open-cluster-management/governance-policy-propagator) 
-- [Policy spec sync](https://github.com/open-cluster-management/governance-policy-spec-sync)
-- [Policy status sync](https://github.com/open-cluster-management/governance-policy-status-sync)
-- [Policy template sync](https://github.com/open-cluster-management/governance-policy-template-sync)
-- Policy controllers: Policy controllers include predefined [out-of-box policy controllers](#out-of-box-policies-and-controllers), or you can [bring your own policy](#bring-your-own-policy).
-
 ![architecture](images/policy-framework-architecture-diagram.jpg)
 
-## Out-of-box policies and controllers
-
-View the following list of predefined policy controllers that are offered with Red Hat Advanced Cluster Management:
-
-- [Configuration policy controller](https://github.com/open-cluster-management/config-policy-controller)
-- [Certificate policy controller](https://github.com/open-cluster-management/cert-policy-controller)
-- [IAM policy controller](https://github.com/open-cluster-management/iam-policy-controller)
-
-## Bring your own policy
+The governance policy framework consists of following components:
+
+- [Governance dashboard](https://github.com/open-cluster-management/grc-ui) -- UI
+- Govenance policy framework -- A framework to distribute various supported policies to managed cluster and collect results back to hub. It consists of following components:
+    - [Policy propagator](https://github.com/open-cluster-management/governance-policy-propagator) 
+    - [Policy spec sync](https://github.com/open-cluster-management/governance-policy-spec-sync)
+    - [Policy status sync](https://github.com/open-cluster-management/governance-policy-status-sync)
+    - [Policy template sync](https://github.com/open-cluster-management/governance-policy-template-sync)
+- Policy controllers -- Policy engines running on managed cluster to evaluate policy rules distributed by the policy framework and generate results.
```

Comment:
I suggest to update "Policy engines running on managed cluster to evaluate policy rules distributed by the policy framework and generate results."

TO 

Policy engines that run on managed clusters to evaluate policy rules distributed by the policy framework and generate results.